mentat/code_file_manager.py

```diff
@@ -69,10 +67,12 @@ async def write_changes_to_files(
     ):
         session_context = SESSION_CONTEXT.get()
         stream = session_context.stream
-        git_root = session_context.git_root
 
         for file_edit in file_edits:
-            rel_path = Path(os.path.relpath(file_edit.file_path, git_root))
+            rel_path = None
+            if file_edit.file_path.is_relative_to(session_context.cwd):
+                rel_path = file_edit.file_path.relative_to(session_context.cwd)
```

Comment:
```suggestion
                rel_path = file_edit.file_path.relative_to(session_context.cwd)
            else:
                rel_path = file_edit.file_path
```
This saves a lot of ink below and makes it a bit clearer what's going on. Since it's only used for displaying, maybe display_path would be better?

Reply:
I agree; the rel_path is only used for printing. What if we just call it 'display_path' and use the full path if it isn't relative to cwd?

Reply:
Sg, will fix this